packages/block-library/src/embed/embed-placeholder.js

```diff
@@ -2,13 +2,16 @@
  * WordPress dependencies
  */
 import { __, _x } from '@wordpress/i18n';
-import { Button, Placeholder } from '@wordpress/components';
+import { Button, Placeholder, ExternalLink } from '@wordpress/components';
 import { BlockIcon } from '@wordpress/block-editor';
 
 const EmbedPlaceholder = ( props ) => {
 	const { icon, label, value, onSubmit, onChange, cannotEmbed, fallback, tryAgain } = props;
 	return (
 		<Placeholder icon={ <BlockIcon icon={ icon } showColors /> } label={ label } className="wp-block-embed">
+			<div className="components-placeholder__instructions">
+				{ __( 'Paste a link to the content you want to display on your site.' ) }
```

Comment:
I think you should be able to pass this text straight into the `Placeholder` component as a prop instead of creating a separate div:
https://github.com/WordPress/gutenberg/blob/0cc2e02811b3fc64e5365927d16a105debc29895/packages/components/src/placeholder/index.js#L33

Something like this would work:
```jsx
<Placeholder
    icon={ <BlockIcon icon={ icon } showColors /> } 
    label={ label } 
    className="wp-block-embed"
    instructions={ __( 'Paste a link to the content you want to display on your site.' ) }
/>

Reply:
Updated!